features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/tracking/ActivityLifecycleTrackingStrategy.kt

```diff
@@ -89,7 +91,17 @@ abstract class ActivityLifecycleTrackingStrategy :
 
     @MainThread
     override fun onActivityCreated(activity: Activity, savedInstanceState: Bundle?) {
-        // No Op
+        val intent = activity.intent
+        val extras = intent.extras
+        val testId = extras?.getString("_dd.synthetics.test_id")
```

Comment:
I am missing the context here...who's going to add these in the extras ? are we going to add this every time we run a synthetic test as runtime arguments ?

Reply:
The synthetics team is the one adding those